docs/APISocketOptions.md

```diff
@@ -877,7 +877,19 @@ In the following cases:
 the connection will be rejected with `SRT_REJ_FILTER` code.
 
 In case of the builtin `fec` filter, the mandatory parameter is `cols`, all
-others have their default values.
+others have their default values. For example, the configuration specified
+as `fec,cols:10` is `fec,rows:1,cols:10,arq:onreq,layout:even`.
+
+Examples for the builtin `fec` filter:
```

Comment:
built-in

Reply:
Then this should be better changed across the docs.
```
> grep builtin *.md
packet-filtering-and-fec.md:their own filters. SRT provides also one builtin filter named "fec". This
packet-filtering-and-fec.md:* alter the packet before inserting it into the SRT channel (the builtin "fec"
```